Demos/DatAgora_PartDieu/API_Enhanced_city-context/web_api.py

```diff
@@ -0,0 +1,419 @@
+#!/usr/bin/env python3
```

Comment:
Identical to UD-Serv.VCityTeam/API_Enhanced_City/api/web_api.py
(up to an added comment)
Remove this file and adapt the Dockerfile accordingly (remove one line)